### PR TITLE
Ceph RGW nfs user modify to specific defined access key and secret key

### PR DESCRIPTION
**Pull request by radut**
RGW NFS user should update with the keys from group vars when those are specified

### CODE REVIEW

File: roles/ceph-nfs/tasks/create_rgw_nfs_user.yml [hunk 1]
```diff
@@ -23,6 +23,16 @@
     - nfs_obj_gw | bool
     - rgwuser_exists.get('rc', 1) != 0
 
+
+- name: Modify rgw nfs user to use specific keys when those are defined
```
Comment (guits) [inline roles/ceph-nfs/tasks/create_rgw_nfs_user.yml]:
```suggestion
- name: modify rgw nfs user to use specific keys when those are defined
```

File: roles/ceph-nfs/tasks/create_rgw_nfs_user.yml [hunk 2]
```diff
@@ -23,6 +23,16 @@
...
+  command: "{{ docker_exec_cmd_nfs | default('') }} radosgw-admin --cluster {{ cluster }} user modify --uid={{ ceph_nfs_rgw_user }} --access-key={{ ceph_nfs_rgw_access_key }} --secret-key={{ ceph_nfs_rgw_secret_key }}"
```
Comment (guits) [inline roles/ceph-nfs/tasks/create_rgw_nfs_user.yml]:
```suggestion
  command: "{{ container_exec_cmd_nfs | default('') }} radosgw-admin --cluster {{ cluster }} user modify --uid={{ ceph_nfs_rgw_user }} --access-key={{ ceph_nfs_rgw_access_key }} --secret-key={{ ceph_nfs_rgw_secret_key }}"
```

**Comment by guits** [comment]
@radut please, read https://github.com/ceph/ceph-ansible/blob/master/CONTRIBUTING.md#commit-guidelines

**Comment by radut** [comment]
Up

**Comment by guits** [comment]
@radut you are still missing a commit message

**Comment by radut** [comment]
Are you reffering to body for commit message ?

**Comment by guits** [comment]
> Are you reffering to body for commit message ?

yes, commit title should be brief whereas body should give more detail.
ie:

```
rgw: support specific keys for rgw nfs user

This brings the possibility to modify the rgw nfs user to use specific keys
when those are defined.
```

**Comment by guits** [comment]
jenkins test centos-non_container-lvm_batch

**Comment by radut** [comment]
Thank you for the suggetion. Updated

**Comment by guits** [comment]
@radut sorry but I put you on the wrong path I think :grin: 

`rgw: support specific keys for rgw nfs user` -> `nfs: support specific keys for rgw nfs user`

**Comment by radut** [comment]
Up

**Comment by guits** [comment]
jenkins test centos-container-collocation

**Comment by guits** [comment]
jenkins test centos-container-lvm_osds

**Comment by guits** [comment]
jenkins test centos-container-update

**Comment by guits** [comment]
jenkins test centos-container-purge

**Comment by guits** [comment]
jenkins test centos-container-podman

File: roles/ceph-nfs/tasks/create_rgw_nfs_user.yml [hunk 3]
```diff
@@ -23,6 +23,16 @@
     - nfs_obj_gw | bool
     - rgwuser_exists.get('rc', 1) != 0
 
+
+- name: modify rgw nfs user to use specific keys when those are defined
+  command: "{{ container_exec_cmd_nfs | default('') }} radosgw-admin --cluster {{ cluster }} user modify --uid={{ ceph_nfs_rgw_user }} --access-key={{ ceph_nfs_rgw_access_key }} --secret-key={{ ceph_nfs_rgw_secret_key }}"
+  delegate_to: "{{ groups[mon_group_name][0] }}"
+  when:
+    - nfs_obj_gw
```
Comment (dsavineau) [inline roles/ceph-nfs/tasks/create_rgw_nfs_user.yml]: Could you add `| bool` on that condition ?

**Comment by radut** [comment]
sure

**Comment by dsavineau** [comment]
jenkins test centos-container-podman

**Comment by guits** [comment]
jenkins test centos-container-podman

**Pull request merged**